SubmittingPatches-backports.rst

```diff
@@ -0,0 +1,344 @@
+* the commit message generated by ``git cherry-pick -x`` must not be modified, except to add a "Conflicts" section
+* the "Conflicts" section must mention all files where changes had to made manually (not just conflicts flagged by git)
+* the "Conflicts" section should also describe the manual changes that were made
+* if a change is to be backported to multiple stable branches, a tracker issue is needed
+
+For more information on that last point, see the `Tracker workflow`_ section.
+
+.. contents::
+   :depth: 3
+
+
+Tracker workflow
+----------------
+
+Any change that is to be backported to multiple stable branches should have
+a tracker issue associated with it. If there isn't one yet, open one at
+https://tracker.ceph.com. This issue should be in the "Bug" or "Feature"
+trackers of the relevant subproject under the "Ceph" parent project (or
+in the "Ceph" project itself if none of the subprojects are a good fit).
+The stable branches to which the master changes are to be cherry-picked should
+be listed in the "Backport" field. For example::
+
+    Backport: mimic, nautilus
+
+Once the PR targeting master is open, add the PR number assigned by GitHub to
+the tracker issue. For example, if the PR number is 99999::
+
+    Pull request ID: 99999
+
+If you do not have sufficient permissions to modify any field of the tracker
+issue, just add a comment describing what changes you would like to make.
+Someone with permissions will make the necessary modifications on your behalf.
+
+Once the master PR has been merged, after checking that the change really needs
+to be backported and the Backport field has been populated, make sure the
+tracker issue's Status field gets changed to "Pending Backport".
+
+For straightforward backports, that's all that you (as the developer of the fix)
+need to do. Volunteers from the `Stable Releases and Backports team`_ will
+proceed to create Backport issues to track the necessary backports and stage the
+backports by opening GitHub PRs with the cherry-picks. If you don't want to
+wait, and provided you have sufficient permissions at https://tracker.ceph.com,
+you can open the Backport tracker issues and stage the backport yourself. In
+that case, read on.
+
+
+Creating Backport tracker issues
+--------------------------------
+
+Before we can start, the master tracker issue needs to exist and be in the right
+state. First, the master issue's Backport field should be populated. For example::
+
+    Backport: mimic, nautilus
+
+Second, the master PR should be in the "Pull request ID" field, e.g.::
+
+    Pull request ID: 99999
+
+Third, the master PR needs to have been merged already.
+
+Fourth, the master tracker issue's status should be set to "Pending Backport"::
+
+    Status: Pending Backport
+
+Once all of the above has been completed, issues can be created in the Backport
+tracker for tracking the backporting work. In our example, above, two issues
```

Comment:
```suggestion
tracker for tracking the backporting work. In the above example, two issues
```

Reply:
@tchaikov "In the example above," <- would that be OK for you?

(Since "above" is not an adjective, strictly speaking it should not go before the noun, though I'm sure you can find examples of such usage on the net.)